### PR TITLE
Phan: Remove wpcomsh stubs from masterbar

### PR DESCRIPTION
**Pull request by rcrdortiz**
## Proposed changes:

* Removed loading wpcomsh stubs since they're now included in the global config.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Open build Phan step.
* There should be no errors

### CODE REVIEW

File: projects/packages/masterbar/.phan/config.php [hunk 1]
```diff
@@ -30,8 +30,6 @@
 			__DIR__ . '/../../../plugins/jetpack/modules/notes.php',
 			__DIR__ . '/../../../plugins/jetpack/modules/scan/class-admin-bar-notice.php',
 			__DIR__ . '/../../../plugins/jetpack/modules/stats.php',
-			__DIR__ . '/../../../plugins/wpcomsh/feature-plugins/masterbar.php',           // function wpcomsh_is_site_sticker_active
-			__DIR__ . '/../../../plugins/wpcomsh/private-site/private-site.php',           // function site_is_private
```
Comment (anomiex) [inline projects/packages/masterbar/.phan/config.php:34]: Only the first one is, not the second.

**Review by anomiex — COMMENTED** [review]
This one too:
https://github.com/Automattic/jetpack/blob/7e75c81edcbcb08897c56ff638c241f7bfbed717/projects/plugins/jetpack/.phan/config.php#L39

File: projects/packages/masterbar/.phan/.config.php.un~
Comment (anomiex) [inline projects/packages/masterbar/.phan/.config.php.un~:1]: Editor file accidentally committed?

File: projects/packages/masterbar/.phan/config.php~
Comment (anomiex) [inline projects/packages/masterbar/.phan/config.php~:1]: Editor file accidentally committed?

File: projects/plugins/jetpack/.phan/.config.php.un~
Comment (anomiex) [inline projects/plugins/jetpack/.phan/.config.php.un~:1]: Editor file accidentally committed?

File: projects/plugins/jetpack/.phan/config.php~
Comment (anomiex) [inline projects/plugins/jetpack/.phan/config.php~:1]: Editor file accidentally committed?

File: projects/packages/masterbar/.phan/config.php (hunk 1 above)
Comment (rcrdortiz) [reply projects/packages/masterbar/.phan/config.php:34]: Restored.

File: projects/packages/masterbar/.phan/.config.php.un~
Comment (rcrdortiz) [reply projects/packages/masterbar/.phan/.config.php.un~:1]: Yes, removed.

File: projects/plugins/jetpack/.phan/.config.php.un~
Comment (rcrdortiz) [reply projects/plugins/jetpack/.phan/.config.php.un~:1]: Yes, removed

File: projects/plugins/jetpack/.phan/config.php~
Comment (rcrdortiz) [reply projects/plugins/jetpack/.phan/config.php~:1]: Yes, removed

File: projects/packages/masterbar/.phan/config.php~
Comment (rcrdortiz) [reply projects/packages/masterbar/.phan/config.php~:1]: Yes, removed

File: projects/packages/masterbar/changelog/remove-wpcomsh-phan-stubs-from-masterbar [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: removed
+
+Removed loading wpcomsh phan stubs from the Masterbar, these stubs are already included globally now'
```
Comment (anomiex) [inline projects/packages/masterbar/changelog/remove-wpcomsh-phan-stubs-from-masterbar]: This doesn't really affect the package at all, just CI checks. So I'd do like this
```suggestion
Significance: patch
Type: removed
Comment: Removed loading wpcomsh phan stubs from the Masterbar, these stubs are already included globally now'


```
That'll then want a `tools/fixup-project-versions.sh` run to bump the versions back down. That might also allow for reverting the version bumps from wpcomsh and mu-wpcom-plugin.

File: projects/plugins/mu-wpcom-plugin/changelog/remove-wpcomsh-phan-stubs-from-masterbar
Comment (anomiex) [inline projects/plugins/mu-wpcom-plugin/changelog/remove-wpcomsh-phan-stubs-from-masterbar:1]: This one seems redundant now too.

File: projects/plugins/wpcomsh/changelog/remove-wpcomsh-phan-stubs-from-masterbar
Comment (anomiex) [inline projects/plugins/wpcomsh/changelog/remove-wpcomsh-phan-stubs-from-masterbar:1]: This one seems redundant now too.

File: projects/packages/masterbar/changelog/remove-wpcomsh-phan-stubs-from-masterbar [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: removed
```
Comment (rcrdortiz) [inline projects/packages/masterbar/changelog/remove-wpcomsh-phan-stubs-from-masterbar:2]: This is not a valid type.

**Review by anomiex — APPROVED** [review]
LGTM, thanks!

**Pull request merged**